src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static Task Timebox(this IEnumerable<Task> tasks, TimeSpan timeoutAfter,
 
                     if (t.IsFaulted && t.Exception != null)
                     {
-                        tcs.TrySetException(t.Exception.InnerException);
+                        tcs.TrySetException(t.Exception.GetBaseException());
```

Comment:
It gives you the inner most exception which is what we actually wanted instead of just relying on InnerException. If inner exception is again an aggregate this call walks to the exception and unpacks it